src/frontend/Sider/Personoversikt/Behandlingsoversikt/BehandlingOversikt.tsx

```diff
@@ -17,21 +25,89 @@ const Behandlingsoversikt: React.FC<{ fagsakPersonId: string }> = ({ fagsakPerso
+                type: behandling.type,
+                behandlingsårsak: behandling.behandlingsårsak,
+                status: behandling.status,
+                vedtaksdato: behandling.vedtaksdato,
+                resultat: behandling.resultat,
+            };
+        });
+        return tabellBehandlinger ? tabellBehandlinger : [];
+    };
+
+    const mapKlagesakRessursTilTabellBehandling = (
+        klageBehandlinger: KlageBehandling[] | undefined
+    ): TabellBehandling[] => {
+        const tabellBehandlinger = klageBehandlinger?.map((klageBehandling) => {
+            return {
+                id: klageBehandling.id,
+                opprettet: klageBehandling.opprettet,
+                type: BehandlingType.KLAGE,
+                behandlingsårsak: klageBehandling.årsak,
+                status: klageBehandling.status,
+                vedtaksdato: klageBehandling.vedtaksdato,
+                resultat: klageBehandling.resultat,
+            };
+        });
+        return tabellBehandlinger ? tabellBehandlinger : [];
+    };
+
+    const feilHaandteringAvTabellBehandlinger = (
+        fagsakPersonRessurs: Ressurs<FagsakPersonMedBehandlinger>,
+        klagebehandlingerRessurs: Ressurs<Klagebehandlinger>
+    ): TabellBehandling[] => {
+        let tabellBehandlingerFraFagsak: TabellBehandling[] = [];
+        if (fagsakPersonRessurs.status === RessursStatus.SUKSESS) {
+            tabellBehandlingerFraFagsak = mapFagsakPersonRessursTilTabellBehandling(
+                fagsakPersonRessurs.data.tilsynBarn?.behandlinger
+            );
+        }
+
+        let tabellBehandlingerFraKlagesaker: TabellBehandling[] = [];
+        if (klagebehandlingerRessurs.status === RessursStatus.SUKSESS) {
+            tabellBehandlingerFraKlagesaker = mapKlagesakRessursTilTabellBehandling(
+                klagebehandlingerRessurs.data.barnetilsyn
+            );
+        }
+
+        return tabellBehandlingerFraFagsak
+            .concat(tabellBehandlingerFraKlagesaker)
+            .sort(sorterBehandlinger);
+    };
+
+    const tabellBehandlinger: TabellBehandling[] = feilHaandteringAvTabellBehandlinger(
+        fagsakPerson,
+        klagebehandlinger
+    );
+
     return (
-        <DataViewer response={{ fagsakPerson, klagebehandlinger }}>
-            {({ fagsakPerson, klagebehandlinger }) => (
-                <>
-                    {fagsakPerson.tilsynBarn && (
-                        <FagsakOversikt
-                            fagsak={fagsakPerson.tilsynBarn}
-                            klagebehandlinger={klagebehandlinger.barnetilsyn}
-                            hentKlagebehandlinger={rekjørHentKlagebehandlinger}
-                            hentBehandlinger={rekjørHentBehandlinger}
-                        />
-                    )}
-                </>
-            )}
-        </DataViewer>
+        <>
+            <DataViewer response={{ klagebehandlinger }}>{() => <></>}</DataViewer>
```

Comment:
Antar dette er for å sende feilmelding hvis klagessaker ikke kan hentes? Det ser jo unektelig litt rart ut, så legg gjerne på en kommentar 🙏

Reply:
Fixed i endringer